{"original_position": 14, "diff_hunk": "@@ -18,15 +18,18 @@ def load_geoip_database():\n \n     try:\n         geoip_reader = Reader(settings.GEOIP2_DATABASE)\n-    except IOError:\n+    except IOError as e:", "body_text": "Oops, this was a part of my debugging (I logged the error). I'll fix that.", "pull_request_url": "https://api.github.com/repos/mozilla/normandy/pulls/102", "url": "https://api.github.com/repos/mozilla/normandy/pulls/comments/58919289", "created_at": "2016-04-07T18:16:37Z", "author_association": "MEMBER", "body": "Oops, this was a part of my debugging (I logged the error). I'll fix that.\n", "updated_at": "2016-04-07T20:58:24Z", "html_url": "https://github.com/mozilla/normandy/pull/102#discussion_r58919289", "pull_request_review_id": null, "_links": {"self": {"href": "https://api.github.com/repos/mozilla/normandy/pulls/comments/58919289"}, "html": {"href": "https://github.com/mozilla/normandy/pull/102#discussion_r58919289"}, "pull_request": {"href": "https://api.github.com/repos/mozilla/normandy/pulls/102"}}, "commit_id": "eccaab0df742f1a29ddc2cfe2f572390d7472258", "user": {"following_url": "https://api.github.com/users/mythmon/following{/other_user}", "events_url": "https://api.github.com/users/mythmon/events{/privacy}", "organizations_url": "https://api.github.com/users/mythmon/orgs", "url": "https://api.github.com/users/mythmon", "gists_url": "https://api.github.com/users/mythmon/gists{/gist_id}", "html_url": "https://github.com/mythmon", "subscriptions_url": "https://api.github.com/users/mythmon/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/305049?v=4", "repos_url": "https://api.github.com/users/mythmon/repos", "received_events_url": "https://api.github.com/users/mythmon/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/mythmon/starred{/owner}{/repo}", "site_admin": false, "login": "mythmon", "type": "User", "id": 305049, "followers_url": "https://api.github.com/users/mythmon/followers"}, "position": null, "path": "normandy/classifier/geolocation.py", "body_html": "<p>Oops, this was a part of my debugging (I logged the error). I'll fix that.</p>", "original_commit_id": "1ce3684025bb9e76c9e713a491a8c4dc09c7f7c3", "id": 58919289}